js/src/components/Match.vue

```diff
@@ -39,7 +41,9 @@ export default {
   data () {
     return {
       match: new Match(),
-      tournament: new Tournament()
+      tournament: new Tournament(),
+      user: this.$root.user,
+      levels: levels,
```

Comment:
@frontierpsycho: Is this an acceptable way of adding the `levels` constants into the apps? I wanted to find a way where we didn't have to do that everywhere but they would be accessible anywhere, but I was not able to!